pages/attacks/XSS_in_Converting_File_Content_to_Text.md

```diff
@@ -20,7 +20,36 @@ its output may result in execution of JavaScript code. This vulnerability has be
 verified by testing some services that translate text from photos and convert
 photos to text. This same process may apply to other vulnerabilities as well!
 
+## Attack Scenario
+
+An attacker can use an image that contains an XSS payload to execute JavaScript
+code on the website. The following descriptions will explain the different stages
+of this scenario and how the XSS attack unfolds within this process.
+
+![Attack Scenario](../assets/images/attacks/XSS_in_Converting_File_Content_to_Text_Attack_Scenario.png)
+    
+Image Upload:<br>
+- An attacker uploads an image containing an XSS payload to the website.
+
+OCR Execution:<br>
+- The OCR system identifies and analyzes the XSS payload.
```

Comment:
This seems to imply that the attack is recognized but I don't think that's actually what you mean. Maybe generic use the statement to "text" instead of payload or something like that...